src/xdp/trn_transit_xdp_maps.h

```diff
@@ -40,6 +41,24 @@ struct bpf_map_def SEC("maps") jmp_table = {
 };
 BPF_ANNOTATE_KV_PAIR(jmp_table, __u32, __u32);
 
+struct bpf_map_def SEC("maps") dfts_map = {
+	.type = BPF_MAP_TYPE_HASH,
+	.key_size = sizeof(struct zeta_key_t),
+	.value_size = sizeof(struct dft_t),
+	.max_entries = 1,
+	.map_flags = 0,
+};
+BPF_ANNOTATE_KV_PAIR(dfts_map, struct zeta_key_t, struct dft_t);
+
+struct bpf_map_def SEC("maps") ftns_map = {
+	.type = BPF_MAP_TYPE_HASH,
+	.key_size = sizeof(struct zeta_key_t),
```

Comment:
same comment as above, shouldn't the key be just ftn id?

Reply:
Currently the zeta_key_t struct only contains a u32 id field.
I don't think a chain/ftn/dft needs any other identifier aside from an ID.
Would it be better to just strip off the struct altogether and just use a u32?

Reply:
agree